ui/app/components/json-editor.js

```diff
@@ -1,4 +1,21 @@
-import Component from '@ember/component';
```

Comment:
only change is glimmerized this component. I was originally here thinking I'd modify this component. In the end I don't use this component, but played boy scout and "left it better than I found it."